packages/orbit-components/src/List/ListItem/index.tsx

```diff
@@ -5,7 +5,7 @@ import * as React from "react";
 import CircleSmall from "../../icons/CircleSmall";
 import type { Props } from "./types";
 
-const ListItem = ({ label, children, icon = <CircleSmall />, dataTest }: Props) => {
+const ListItem = ({ label, children, icon = <CircleSmall />, dataTest, ariaHidden }: Props) => {
```

Comment:
When you squash it, I believe this commit should be a feat, as we're adding a new prop.

Reply:
No relevant if we follow https://github.com/kiwicom/orbit/pull/4678/commits/acc6b081a27198007384860b9bde7f254ebb3325#r2007821216